docs/en/remote-flow.rst

```diff
@@ -202,12 +202,8 @@ Conversely, in the **Same Device Flow**, the Relying Party uses an HTTP response
             "id": "wallet attestation",
             "format": "dc+sd-jwt",
             "meta": {
-              "vct_values": ["https://itwallet.registry.example.it/WalletAttestation"]
-            },
-            "claims": [
-              {"path": ["wallet_link"]},
-              {"path": ["wallet_name"]}
-            ]
+              "vct_values": ["https://trust-registry.it-wallet.example.it/v1.0/WalletAttestation"]
```

Comment:
please say if this RL is case sensitive

I'd go for a case insentive approach within HTTPD configurations